x-pack/plugins/secops/public/components/timeline/body/renderers/column_renderer.ts

```diff
@@ -5,8 +5,9 @@
  */
 
 import { Ecs } from '../../../../graphql/types';
+import { EcsField } from '../../../../lib/ecs';
 
 export interface ColumnRenderer {
   isInstance: (columnName: string, data: Ecs) => boolean;
-  renderColumn: (columnName: string, data: Ecs) => React.ReactNode;
+  renderColumn: (columnName: string, data: Ecs, field: EcsField | undefined) => React.ReactNode;
```

Comment:
Was this suppose to be `field?: EcsField)` or is there something extra with the `undefined` I'm not seeing.